cmd/eskip/readclient.go

```diff
@@ -0,0 +1,127 @@
+package main
+
+import (
+	"github.com/zalando/skipper/eskip"
+	"github.com/zalando/skipper/eskipfile"
+	etcdclient "github.com/zalando/skipper/etcd"
+	innkeeperclient "github.com/zalando/skipper/innkeeper"
+	"io"
+	"io/ioutil"
+	"os"
+)
+
+type ReadClient interface {
+	LoadAndParseAll() ([]*eskip.RouteInfo, error)
+
+	LoadUpdate() (eskip.RouteList, []string, error)
```

Comment:
please, just don't replace it yet. Go interfaces are quite different than java interfaces, and we should discuss it first.

In general, i think it would be nicer adding small features and doing full refactorings in separate pull requests. The design considerations for the latter could be discussed before making so big changes, too.